### DISCUSSION TITLE
Setting fields to be ignored by translations

### DISCUSSION THREAD

**Discussion by SalahAdDin** [directus/directus#14860] [Archive]
Hello here,

Sometimes we need to localize/translate a collection for not all its fields but only for a set of them.

Is it possible right now?

Thank you.

**Comment by rijkvanzanten** [comment]
Yes. When you configure translations, you explicitly add fields that are in the translations UI.

**Reply by SalahAdDin** [reply]
how can we do that? i couldn't figure out that option.

**Reply by rijkvanzanten** [reply]
What have you tried so far?

**Reply by SalahAdDin** [reply]
I followed the tutorial and added a new translation field:
![Screen Shot 2022-08-20 at 18 18 26](https://user-images.githubusercontent.com/5034215/185754063-a52aab5e-b7ee-4f93-b48e-b8c77a36671c.png)

Now, looking at the field, I try to find the options for choosing only the fields I want to translate:
https://user-images.githubusercontent.com/5034215/185754233-682287d5-d9d4-4a1c-b917-5f0e0f40104f.mov
I can't find anything related to it.

And finally, I just want to translate the title/slug and nothing more.

How will it work then? I can't find any visible way to choose only the fields I want to translate.

Thank you.
![Screen Shot 2022-08-20 at 18 22 04](https://user-images.githubusercontent.com/5034215/185754260-69f77c0d-ed91-4549-9a62-9ba1f306179e.png)

**Reply by azrikahar** [reply]
> And finally, I just want to translate the title/slug and nothing more.

`title` and/or `slug` field will need to be created in the junction table _after_ you created the `translations` field. For example, once you created the `translations` field on a collection called `articles`, the newly created junction table should be called `articles_translations`. Any fields created in that junction table will be the fields that can be translated.

> I followed the tutorial

Out of curiosity, which tutorial were you following? Could it be https://docs.directus.io/configuration/data-model/relationships.html#translations-o2m, or somewhere else?

**Reply by SalahAdDin** [reply]
> > And finally, I just want to translate the title/slug and nothing more.
> 
> `title` and/or `slug` field will need to be created in the junction table _after_ you created the `translations` field. For example, once you created the `translations` field on a collection called `articles`, the newly created junction table should be called `articles_translations`. Any fields created in that junction table will be the fields that can be translated.
> 
> > I followed the tutorial
> 
> Out of curiosity, which tutorial were you following? Could it be https://docs.directus.io/configuration/data-model/relationships.html#translations-o2m, or somewhere else?

Actually, It was a Tutorial on YouTube, unfortunately, the Directus channel does not have new tutorials or feature videos, it is like abandoned.
